docs/digital_analog_qc/analog-qcl.md

```diff
@@ -0,0 +1,128 @@
+Analog blocks can be parametrized in the usual Qadence manner. Like any other parameters,
+they can be optimized. The next snippet examplifies the creation of an analog and parameterized ansatz
+to fit a simple function. First, define a register and feature map block:
+
+```python exec="on" source="material-block" session="qcl"
+from qadence import Register, FeatureParameter, chain
+from qadence import AnalogRX, AnalogRY, AnalogRZ, wait
+
+# Line register
+n_qubits = 2
+dx = 8.0
+coordinates = [(i*dx, 0) for i in range(n_qubits)]
+register = Register.from_coordinates(coordinates)
+
+# The input feature parameter
+x = FeatureParameter("x")
```

Comment:
Would it make sense to rename it something that resembles a rotation angle ? Ie. `theta` ? So as not to confuse with the coordinates.

Reply:
I think I would rather keep it as "x" in the sense of "learning f(x)". It does get encoded into an angle, but the meaning of that feature is not necessarily an angle.

Reply:
Right OK. I would still try to disambiguate the role of that variable in the comment if possible.